Simple-Store.md

```diff
@@ -0,0 +1,87 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [Use Simple Perstistence Store](#use-simple-perstistence-store)
+- [Overview](#overview)
+  - [Use Simple Perstistence Store for workorders in Cloud App](#use-simple-perstistence-store-for-workorders-in-cloud-app)
+    - [Create Simple Store in your Cloud App](#create-simple-store-in-your-cloud-app)
+    - [Registering CRUD topics for the workorders](#registering-crud-topics-for-the-workorders)
+    - [Create list of of workorders to be saved into Simple Store](#create-list-of-of-workorders-to-be-saved-into-simple-store)
+    - [Reset the workorder dates to today to allow viewing workorders in the scheduler](#reset-the-workorder-dates-to-today-to-allow-viewing-workorders-in-the-scheduler)
+    - [Publishing a create topic to create the workorders](#publishing-a-create-topic-to-create-the-workorders)
+  - [Option to switch between array store and persistence](#option-to-switch-between-array-store-and-persistence)
+  - [Removing data from the data browser in the deployed raincatcher-demo-cloud and raincatcher-demo-auth applications](#removing-data-from-the-data-browser-in-the-deployed-raincatcher-demo-cloud-and-raincatcher-demo-auth-applications)
+  - [Resetting a persistent storage raincatcher project](#resetting-a-persistent-storage-raincatcher-project)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Use Simple Perstistence Store
+
+# Overview
+- This is a new module raincatcher-store-simple that will
+    - use current functionality in the mediator ArrayStore 
+    - use the $fh.db cloud API to store data in a mongo database
+-  The cloud applications can specify a `WFM_USE_MEMORY_STORE` environment variable 
+    - allow users to control whether the memory store or the MongoDB store is used.
+
+## Use Simple Perstistence Store for workorders in Cloud App
+
+### Create Simple Store in your Cloud App
+```js
+const SimpleStore = require('fh-wfm-simple-store');
+const simpleStore = new SimpleStore('workorders');
+simpleStore.init({/* init with data */}).then(function() {
+  // store is available for use
+});
+```
+### Registering CRUD topics for the workorders
+```js
+simpleStore.listen('cloud:', mediator);
+```
+
+### Create list of of workorders to be saved into Simple Store 
+```js
```

Comment:
Might want to move this before the `Create Simple Store in your Cloud App` section to identify the data that the store can be initialised with.